src/cascadia/TerminalSettingsEditor/EditColorScheme.xaml

```diff
@@ -450,6 +450,39 @@
                     </Button>
                 </StackPanel>
             </local:SettingContainer>
+
+            <local:SettingContainer x:Name="SetAsDefaultContainer"
+                                    x:Uid="ColorScheme_SetAsDefault">
+                <Button x:Uid="ColorScheme_SetAsDefaultButton"
+                        Style="{StaticResource BrowseButtonStyle}"
+                        Click="{x:Bind ViewModel.SetAsDefault_Click}"/>

```

Comment:
Idk if this does anything noticeable, but `BrowseButtonStyle` adds a margin of `10,0,0,0`. Might be worth overriding it to `0`?

Reply:
Doesn't seem to be necessary haha